### PR TITLE
[RELEASE] chore: update to v0.3.1

### PR DESCRIPTION
**Pull request by simbleau**
Fix the changelog format of previous releases.

Release version 0.3.1.

### CODE REVIEW

**Review by nicoburns — APPROVED** [review]

**Comment by DJMcNab** [comment]
I've backed this out because of the PR description - making commits which mention people has occasionally backfired, because GitHub likes pinging that user when people make forks/do merges incorrectly, etc.

I've had that a few times in Bevy, and so want to avoid that here.

**Comment by waywardmonkeys** [comment]
And please fix the changelog in a separate PR from doing the release.

File: CHANGELOG.md
[the remark this answers is not in the document]
Comment (simbleau) [reply CHANGELOG.md:1]: done

**Comment by simbleau** [comment]
> And please fix the changelog in a separate PR from doing the release.

> And please fix the changelog in a separate PR from doing the release.

Done, that's now #30

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,17 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
 
+The latest published Vello release is [0.3.1](#031-2024-07-29) which was released on 2024-07-29.
```
Comment (waywardmonkeys) [inline CHANGELOG.md:11]: I think this is supposed to say the `vello_svg` here.

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,10 +8,17 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
 
+The latest published vello_svg release is [0.3.1](#031-2024-07-29) which was released on 2024-07-29.
```
Comment (DJMcNab) [inline CHANGELOG.md:11]:
```suggestion
The latest published Vello SVG release is [0.3.1](#031-2024-07-29) which was released on 2024-07-29.
```
This matches the equivalent file in Vello, which uses the "proper" form.

**Comment by DJMcNab** [comment]
Not approving until #30 lands and is merged with this.

**Review by DJMcNab — APPROVED** [review]
Thanks!

**Pull request merged**